GenderPayGap.Database/Models/Extensions/Organisation.cs

```diff
@@ -324,17 +324,17 @@ public Return GetLatestReturn()
                 .FirstOrDefault();
         }
 
-        //Returns the latest return for the specified accounting year or the latest ever if no accounting year is 
+        // Returns the latest return for the specified accounting year or the latest ever if no accounting year is 
         public Return GetReturn(int year = 0)
         {
             DateTime accountingStartDate = SectorType.GetAccountingStartDate(year);
             return Returns
-                .Where(r => r.Status == ReturnStatuses.Submitted && r.AccountingDate == accountingStartDate)
+                .Where(r => r.Status == ReturnStatuses.Submitted && r.AccountingDate.Year == accountingStartDate.Year)

```

Comment:
Looks all good, just wanted to check the issue was as I cannot see the JIRA page yet 😭